addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5428,12 +5436,12 @@ msgstr ""
 
 #: xbmc\osx\XBMCHelper.cpp
 msgctxt "#13144"
-msgid "Change Apple Remote mode?"
+msgid "Change Apple remote mode?"
 msgstr ""
 
 #: xbmc\osx\XBMCHelper.cpp
 msgctxt "#13145"
-msgid "If you are currently using the Apple Remote to control"
+msgid "If you are currently using the Apple remote to control"
```

Comment:
Marketing spew camel casing is incorrect imo as remote is not capitalizeable in any dictionary.